src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -45,14 +46,18 @@ class Program
             "net",
             "netcore"
         };
-
+        
         protected static ActorSystem TestRunSystem;
         protected static IActorRef SinkCoordinator;
 
         /// <summary>
         /// file output directory
         /// </summary>
         protected static string OutputDirectory;
+        /// <summary>
+        /// Subdirectory to store failed specs logs
+        /// </summary>
+        protected static string FailedSpecsDirectory;
```

Comment:
We need to document this feature and where it's output goes here https://github.com/akkadotnet/akka.net/blob/dev/docs/articles/testing/multi-node-testing.md

Reply:
Maybe that can be a separate issue

Reply:
Let's put it into separate issue. Here is it: https://github.com/akkadotnet/akka.net/issues/4107

Reply:
Sounds good